Jenkinsfile

```diff
@@ -302,8 +299,8 @@ stage('Default build') {
 							install \
 					"} \
 					-Pdist \
-					-Psigtest \
-					-Pjqassistant \
+					${enableDefaultBuildSigtest ? '-Psigtest' : ''} \
+					${enableDefaultBuildJQAssistant ? '-Pjqassistant' : ''} \
```

Comment:
Not sure it matters, but you don't set `enableDefaultBuildSigtest`/`enableDefaultBuildJQAssistant` anywhere. They will always be `false`.

Reply:
Yes I know :). I plan to improve on that to get the info from the default build env but I didn't have the time to figure out how to do it properly and we don't really care for now as it doesn't work with 17 at all.